packages/base/lua/Profiler.lua

```diff
@@ -113,8 +114,14 @@ local createBlock = function(name)
 			self.running = true
 		end,
 		stop = function(self)
+			if not self.running then
```

Comment:
It is always better to have positive ifs. In this case, use `if self.running` and move the three lines after the if to here. Note that the return value is the same, and therefore line 118 is unnecessary.